app/classifier/tasks/survey/choice.cjsx

```diff
@@ -52,7 +52,7 @@ module.exports = React.createClass
     onCancel: Function.prototype
 
   getInitialState: ->
-    answers: {}
+    answers: @props.annotation?.answers || {}
```

Comment:
Looking at the coffeescript for `Choice`, looks like the original practice on PFE was to document props by adding them to `getDefaultProps` so maybe `annotation: { answers: {} }` should go in there after all.